runtime/evm_tracer/src/raw.rs

```diff
@@ -96,6 +96,9 @@ impl RawTracer {
 	/// Consume the tracer and return it alongside the return value of
 	/// the closure.
 	pub fn trace<R, F: FnOnce() -> R>(self, f: F) {
+		evm_gasometer::tracing::enable_tracing(true);
```

Comment:
This isn't missing `evm::tracing::enable_tracing()`, is it?

Reply:
The raw tracer is not listening for `evm` events so it's not required here.